### PR TITLE
Theme easter egg

### PR DESCRIPTION
**Pull request by d4rekanguok**
Hi folks! I finally get around to implement _the_ theme that I wanted to do when I first suggested the feature. It's supposed to be a nod to one of my first experience with Microsoft:

theme-msdos

[reference for youngsters (MS-DOS UI)](https://ilyabirman.net/meanwhile/pictures/nc.png)

But now seeing everyone's improvement on the light/dark theme, I think maybe it's best to leave the theme as light/dark only and maybe have this as an easter egg? (runs `msdos()` in the console)

However this does look a bit like the infamous blue screen of death, so maybe it's not a good idea at all.

### CODE REVIEW

File: docs/theme/theme.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import "msdos/font.css";
```
Comment (d4rekanguok) [inline docs/theme/theme.css]: If we decide to use this theme I'd find a javascript solution so viewers don't have to load unnecessary resources

**Comment by dvdsgl** [comment]
This is great. Why don't we show this theme if people click on the Microsoft logo? That could toggle it.

Also, you've replaced the Microsoft logo with the Windows logo – maybe find a retro Microsoft logo instead?

Maybe unlocking this theme could be the reward for beating the Simon game? (#41)

**Comment by d4rekanguok** [comment]
> Maybe unlocking this theme could be the reward for beating the Simon game?
That'd be neat! @makeupsomething

> Also, you've replaced the Microsoft logo with the Windows logo – maybe find a retro Microsoft logo instead?
Blibbet's the logo that Microsoft used around that time, I couldn't find one with good resolution + free license yet. Will dig harder

**Comment by makeupsomething** [comment]
Yeah thats a pretty cool idea! I should have time to look into that tomorrow 🎉

**Comment by d4rekanguok** [comment]
Hi folks, some updates:

image

This update will also allow adding additional themes by creating a named subfolder in `docs/theme` that contains at least a `theme.css` file. Theme then can be loaded by calling `theme.changeTo(themeName)`.

```
# Theme folder:
docs
└── theme
     └── msdos
          ├── (fonts, images etc.)
          └── theme.css

# Change to theme 'msdos':
theme.changeTo('msdos')
```

We could make it so that when user beat the simon game, add an extra button for msdos down in the theme switcher. What do you think?

**Comment by makeupsomething** [comment]
Sorry for my lateness on this. It's been a busy week 💤 

Any ideas how to go about integrating this? I think I cannot merge this branch into my own fork as they are both remotes. Or am I thinking about it the wrong way? @d4rekanguok @dvdsgl

**Comment by d4rekanguok** [comment]
@makeupsomething I think ideally you won't need to merge this branch, only need to do a theme switch in your code, maybe something like `if (theme && theme.changeTo) theme.changeTo('msdos');` and when this PR is landed it'll work. What do you think?

**Comment by dvdsgl** [comment]
Great work! Maybe someone should just add this to the theme switcher?

**Comment by makeupsomething** [comment]
Nice @d4rekanguok you were right, after this PR was merged I was able to easily switch theme when the user wins. https://github.com/Microsoft/join-dev-design/pull/41/commits/da3f02b5840bf11099f8e3995f10db0669af8a11

File: docs/theme/theme.js [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+  activeTheme: "dark",
+  load: function(theme, cb) {
+    var id = "theme-" + theme;
+    if (!document.getElementById(id)) {
+      var $head = document.getElementsByTagName("head")[0];
+      var $link = document.createElement("link");
+      $link.id = id;
+      $link.rel = "stylesheet";
+      $link.type = "text/css";
+      $link.href = "/theme/" + theme + "/theme.css";
```
Comment (d4rekanguok) [inline docs/theme/theme.js]: @dvdsgl ack, I think I missed a dot here and it breaks on the live site -- so sorry! Could you put in a fix?

Comment (dvdsgl) [reply docs/theme/theme.js]: Please send a PR! I noticed it was broken.

Comment (d4rekanguok) [reply docs/theme/theme.js]: 🙏 I've sent one!

**Pull request merged**